tools/c7n_mailer/tests/test_ldap.py

```diff
@@ -0,0 +1,96 @@
+# Copyright 2016 Capital One Services, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+from common import destroy_local_ldap_cache, get_ldap_lookup, PETER, BILL
+
+
+class MailerLdapTest(unittest.TestCase):
```

Comment:
add a bunch of ldap unit tests, ldap is pretty solidly covered. the ldap3 mock section of the library was helpful!